app/models/similar_occupation_standards.rb

```diff
@@ -8,6 +10,27 @@ def initialize(occupation_standard)
   end
 
   def top_five
-    OccupationStandard.search
+    OccupationStandard.__elasticsearch__.search(query).records
+  end
+
+  private
+
+  def query
+    {
+      size: 5,
```

Comment:
I'm wondering if we should make this a constant, and rename the `top_five` method name to be more generic in case we decide to change the number of results we want to return. Then you could also stub the constant in the spec to be something small and confirm that it returns the correct number of results.

Reply:
done ✅